### PR TITLE
chore: upgrade react/react-dom to v16.7

### PR DESCRIPTION
**Pull request by frogzsj**
BREAKING CHANGE: because it changes min requirements

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @frogzsj, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by frogzsj** [comment]
CLA signed

**Comment by priyajeet** [comment]
Needs to be marked as a BREAKING change

File: package.json [hunk 1]
```diff
@@ -201,9 +201,9 @@
         "properties-parser": "^0.3.1",
         "query-string": "5.1.1",
         "raf": "^3.4.0",
-        "react": "^16.5.1",
+        "react": "^16.7.0",
```
Comment (priyajeet) [inline package.json]: update peer deps too

**Pull request merged**